docs/brainstorms/2026-03-19-improve-gt-setup-for-ai-agent-workflows-brainstorm.md

```
@@ -0,0 +1,206 @@
+## What We're Building
```

markdownlint-cli2 (0.21.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)